release-notes/unreleased/3182-prevent-polling-before-first-message.yml

```diff
@@ -0,0 +1,5 @@
+issue_key: 3182
+show_in_stores: false
+platforms: # relevant platforms, possible values: web, android and ios
+  - web
+en: Improvements to the integreat Chat Support.
```

Comment:
🤔 I think this release note is redundant since it is not shown to our users and does not provide any information to devs or other teams as it is really unspecific. I think it can just be removed.
Generally, `Integreat` should always be uppercased and the feature is just called `Integreat Chat` without `Support`.